server/game/stateWatchers/CardsPlayedThisPhaseWatcher.ts

```diff
@@ -4,11 +4,14 @@ import { StateWatcherRegistrar } from '../core/stateWatcher/StateWatcherRegistra
 import Player from '../core/Player';
 import { TokenOrPlayableCard } from '../core/card/CardTypes';
 import { Card } from '../core/card/Card';
+import { InPlayCard } from '../core/card/baseClasses/InPlayCard';
 
 export interface PlayedCardEntry {
     card: TokenOrPlayableCard;
     inPlayId?: number;
     playedBy: Player;
+    parentCard: InPlayCard;
```

Comment:
These parent card properties should be optional (`parentCard?`)

Reply:
made optional